docs/artifactory/repodelete/help.go

```diff
@@ -0,0 +1,8 @@
+package repodelete
+
+const Description = "Remove a repository from Artifactory."
```

Comment:
Remove a repository... --> Permanently delete a repository with all of its content...